### ISSUE TITLE
Segmentation Fault when Build Phase Script file does not exist

### ISSUE THREAD

**Issue by rm98** [yonaskolb/XcodeGen#997]
When a build phase script file does not exist `XcodeGen` runs into a segfault:
```sh
$ swift run xcodegen --use-cache
Segmentation fault: 11
```

An example for a part of a `project.yml` needed to reproduce the segmentation fault:
```
targets:
  CustomFramework:
    sources:
      - Sources/CustomFramework
    preBuildScripts:
    - path: build_scripts/file-that-does-not-exist.sh
      name: Build Phase Name
```

Versions:
* `XcodeGen` was integrated via SPM, the relevant section from the `Package.resolved`:
```json
{
  "package": "XcodeGen",
  "repositoryURL": "https://github.com/yonaskolb/XcodeGen.git",
  "state": {
    "branch": null,
    "revision": "bb4a3fa2dd720594e47f33cd48cce84fcf9f7066",
    "version": "2.18.0"
  }
}
```
* macOS 11.1 (Big Sur)
* Swift 5.3.2
```sh
$ swift --version
Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)
Target: x86_64-apple-darwin20.2.0
```

**Comment by riley-kohler** [comment]
I was looking into this issue as a potential contribution but was unable to reproduce it. The error I get is
`Spec validation error: Target "Enterprise" has a script "Build Phase Name" which has a path that doesn't exist "build_scripts/file-that-does-not-exist.sh"` which seems accurate. I'm using Swift 5.3 on macOS 10.15 (Catalina) so not sure if that affects reproduction.

**Comment by rm98** [comment]
Hi @riley-kohler, thanks for checking on this!

I'll check tomorrow and will close/attach a sample project if that issue persists for me.

**Comment by rm98** [comment]
Indeed - it seems like this was fixed in `2.19.0` (as I used `2.18.0` when reporting).

Therefore, I will close the ticket.
Again, thanks for checking @riley-kohler!
